brontology/database/connector.py

```diff
@@ -0,0 +1,46 @@
+"""The connector to the neo4j database."""
+
+import os
+from contextlib import contextmanager
+from dataclasses import dataclass
+
+from dotenv import load_dotenv
+from neo4j import GraphDatabase, Driver
+
+load_dotenv()
+
+
+@dataclass
+class _Connector:
+    """Singleton to connect to the neo4j database.
+    Use the `.driver` contextmanager to retrieve a driver for this connection."""
+
+    uri: str
+    username: str
+    password: str
+
+    @contextmanager
+    def driver(self) -> Driver:
```

Comment:
Der Driver ist der wichtigste Teil, das ist wie bei SQL der Cursor, in dem man Transaktionen durchführt